debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerStreamingChangeEventSource.java

```diff
@@ -331,6 +335,9 @@ private void migrateTable(SqlServerPartition partition, final Queue<SqlServerCha
                 new SqlServerSchemaChangeEventEmitter(partition, offsetContext, newTable, tableSchema,
                         SchemaChangeEventType.ALTER));
         newTable.setSourceTable(tableSchema);
+        if (streamingProgressListener.isPresent()) {
```

Comment:
There are `NO_OP` implementations of some of the `Listener` interfaces. It would make sense to use the same approach here.